### PR TITLE
SONARPY-254 Require SonarQube 6.7

### PR DESCRIPTION
**Pull request by pynicolas**

### CODE REVIEW

File: sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java [hunk 1]
```diff
@@ -60,11 +53,7 @@ public void define(Context context) {
   }
 
   private RuleMetadataLoader getRuleMetadataLoader() {
```
Comment (SonarTech) [inline sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonRuleRepository.java]: Code Smell Code Smell: Make "getRuleMetadataLoader" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-254&issues=AWD-dbT4EFfjxkQcPwFu&open=AWD-dbT4EFfjxkQcPwFu)

[ISSUE_KEY]: AWD-dbT4EFfjxkQcPwFu

File: sonar-python-plugin/src/test/java/org/sonar/plugins/python/TestUtils.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.sonar.plugins.python;
+
+public class TestUtils {
```
Comment (alban-auzeill) [inline sonar-python-plugin/src/test/java/org/sonar/plugins/python/TestUtils.java]: what this class is for?

Comment (alban-auzeill) [reply sonar-python-plugin/src/test/java/org/sonar/plugins/python/TestUtils.java]: Ignore the above comment, now the class has at least one method in this commit.

File: sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java [hunk 3]
```diff
@@ -25,7 +25,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.sonar.api.batch.fs.internal.DefaultInputFile;
-import org.sonar.api.batch.fs.internal.FileMetadata;
+import org.sonar.api.batch.fs.internal.TestInputFileBuilder;
```
Comment (alban-auzeill) [inline sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonHighlighterTest.java]: We should not depend on "internal" api.

File: sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonTest.java [hunk 4]
```diff
@@ -22,8 +22,8 @@
 import com.google.common.collect.Maps;
 import java.util.Map;
 import org.junit.Test;
-import org.sonar.api.config.MapSettings;
 import org.sonar.api.config.Settings;
+import org.sonar.api.config.internal.MapSettings;
```
Comment (alban-auzeill) [inline sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonTest.java]: We should not depend on "internal" api.

**Pull request merged**